guides/source/active_record_encryption.md

```diff
@@ -163,152 +186,122 @@ class Article < ApplicationRecord
-class Person
-  encrypts :email_address, deterministic: true, downcase: true
-end
+# config/applicaiton.rb
+config.active_record.encryption.extend_queries = true
 ```
 
-When using `:downcase`, the original case is lost. In some situations, you might want to ignore the case only when querying while also storing the original case. For those situations, you can use the option `:ignore_case`. This requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
+Then, the uniqueness constraint can be specified normally, along with encryption:
 
 ```ruby
-class Label
-  encrypts :name, deterministic: true, ignore_case: true # the content with the original case will be stored in the column `original_name`
+class Person
+  validates :email_address, uniqueness: true
+  encrypts :email_address, deterministic: true, downcase: true
 end
 ```
 
-### Support for Unencrypted Data
-
-To ease migrations of unencrypted data, the library includes the option `config.active_record.encryption.support_unencrypted_data`. When set to `true`:
-
-* Trying to read encrypted attributes that are not encrypted will work normally, without raising any error.
-* Queries with deterministically encrypted attributes will include the "clear text" version of them to support finding both encrypted and unencrypted content. You need to set `config.active_record.encryption.extend_queries = true` to enable this.
-
-**This option is meant to be used during transition periods** while clear data and encrypted data must coexist. Both are set to `false` by default, which is the recommended goal for any application: errors will be raised when working with unencrypted data.
-
-### Support for Previous Encryption Schemes
-
-Changing encryption properties of attributes can break existing data. For example, imagine you want to make a deterministic attribute non-deterministic. If you just change the declaration in the model, reading existing ciphertexts will fail because the encryption method is different now.
-
-To support these situations, you can declare previous encryption schemes that will be used in two scenarios:
+The `extended_queries` configuration also allows combining encrypted and unencrypted data, and when configuring previous encryption schemes.
 
-* When reading encrypted data, Active Record Encryption will try previous encryption schemes if the current scheme doesn't work.
-* When querying deterministic data, it will add ciphertexts using previous schemes so that queries work seamlessly with data encrypted with different schemes. You must set `config.active_record.encryption.extend_queries = true` to enable this.
-
-You can configure previous encryption schemes:
-
-* Globally
-* On a per-attribute basis
-
-#### Global Previous Encryption Schemes
+NOTE: If you want to ignore the case, make sure to use the `downcase` or `ignore_case` option in the `encrypts` declaration. Using the `case_sensitive` option in the validation won't work.
```

Comment:
It was a bit unclear to me this note is for uniqueness.
I think it's also clearer to use colons as a prefix for options here:
Maybe something like:
```suggestion
NOTE: If you want to ignore the case for uniqueness, make sure to use the `:downcase` or `:ignore_case` option in the `encrypts` declaration. Using the `:case_sensitive` option in the validation won't work.
```